app/templates/components/password/forgot-password.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="container">
+  <form>
+
+    {{input name="email" autocapitalize="off" type="text" placeholder='Your email' value=email key-down="keyDown"}}
```

Comment:
Above this we can have a `p` that says:

> Enter your email and we'll send you a link to reset your password

If the user's authenticated, this could even be auto-populated with the user's email!

Don't think `keyDown` is needed here.